### ISSUE TITLE
Echo UI app doesn't start on cloud if the dependencies are not installed locally.

### ISSUE THREAD

**Issue by manskx** [Lightning-AI/LAI-Echo-App#113]
In lightning 1.8 release, it introduced functionality to start an app on the cloud without installing the dependencies locally. But it doesn't work with the Eco app. 

It might be a bug in the framework, more investigation is required. 

How to reproduce ?

```
pip install ligntning
git clone git@github.com:Lightning-AI/LAI-Echo-App.git
cd LAI-Echo-App
lightning run app app.py --cloud

```

Expected behavior:
The app starts on the cloud with a functional UI. (the UI return 502)

**Comment by alecmerdler** [comment]
@manskx I don't believe this is related to dependencies. In the steps to reproduce, you didn't do the necessary steps to build the frontend (see the `README.md`). That could be why the UI is responding with `HTTP 502`.
